index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<head>
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css" />
+  <link
+    rel="stylesheet"
+    type="text/css"
+    href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+  />
+  <!-- Google tag (gtag.js) -->
+  <script
+    async
+    src="https://www.googletagmanager.com/gtag/js?id=G-F55ED7P0L4"
+  ></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag("js", new Date());
+
+    gtag("config", "G-F55ED7P0L4");
+  </script>
+</head>
+<body>
+  <header class="top-bar">
+    <div>
+      <a href="/" class="pglogo">
+        <img src="logo.png" class="logo-icon" alt="로고" />
+      </a>
+      <a href="/login.html" class="login-btn">
+        <img src="loginbt.png" class="login-icon" alt="로그인" />
+      </a>
+    </div>
+  </header>
+
+  <section class="section2">
+    <div class="section2-wrap">
+      <div class="section-text">
+        <h1>일상의 모든 물건을<br />거래해 보세요</h1>
+        <a href="/item">
+          <button>구경하러 가기</button>
+        </a>
+      </div>
+
+      <img src="panda.png" class="panda-icon" alt="판다 " />
+    </div>
+  </section>
+
+  <section class="section3">
+    <img src="section3.png" class="section-img" alt="3" />
+    <div class="section-text">
+      <span>Hot item</span>
+      <h1>인기 상품을 <br />확인해 보세요</h1>
+      <p>가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요</p>
+    </div>
+  </section>
+  <section class="section4">
+    <img src="section4.png" class="section-img" alt="4" />
+    <div class="section-text">
+      <span>Search</span>
+      <h1>구매를 원하는<br />상품을 검색하세요</h1>
+      <p>구매하고 싶은 물품은 검색해서<br />쉽게 찾아보세요</p>
+    </div>
+  </section>
+  <section class="section5">
+    <img src="section5.png" class="section-img" alt="5" />
+    <div class="section-text">
+      <span>Register</span>
+      <h1>판매를 원하는<br />상품을 등록하세요</h1>
+      <p>어떤 물건이든 판매하고 싶은 상품을<br />쉽게 등록하세요</p>
+    </div>
+  </section>
+  <div class="marjin"></div>
+
+  <section class="last-box">
+    <div class="section2-wrap">
+      <div class="section-text">
+        <h1>믿을 수 있는<br />판다마켓 중고 거래</h1>
+      </div>
+
+      <img src="panda2.png" class="panda2-icon" alt="판다 " />
+    </div>
+  </section>
+
+  <div class="footer2">
+    <div class="footer">
```

Comment:
footer2 내부에 footer가 있는 게 어색해보이지 않나요? 스타일을 적용할 때 종종 나타나는 현상인데, 저는 이련 경우 -wrapper, -container, -inner 같은 이름을 활용하는 편입니다.

```suggestion
  <div class="footer-container">
    <div class="footer">
```

```suggestion
  <div class="footer-wrapper">
    <div class="footer">
```

```suggestion
  <div class="footer">
    <div class="footer-inner">
```